provider/resources.go

```diff
@@ -2469,6 +2469,34 @@ func ProviderFromMeta(metaInfo *tfbridge.MetadataInfo) *tfbridge.ProviderInfo {
 						AltTypes:  []tokens.Type{awsType(iamMod, "documents", "PolicyDocument")},
 						Transform: tfbridge.TransformJSONDocument,
 					},
+					"inline_policy": {
+						// inline_policy is an array of policy objects. It is allowed to provided an empty list
+						//   inlinePolicies: []
+						// or a list with empty objects
+						//   inlinePolicies: [{}]
+						// In both cases the role will be created _without_ any inline policies attached.
+						// If a policy is provided, then both the `policy` and the `name` fields are required.
+						// If one is provided and the other is not, then no error will be thrown and no inline policy
+						// will be created.
+						Transform: func(pv resource.PropertyValue) (resource.PropertyValue, error) {
+							inlinePolicy := []resource.PropertyValue{}
+							for _, value := range pv.ArrayValue() {
```

Comment:
`pv` could be a computed value, which would cause this transform to panic. We always need to do a type check (`.IsArray()`) before casting (`.ArrayValue()`) since any value could be computed.

Reply:
Done!